### PR TITLE
fix(lint): revive CI

### PR DESCRIPTION
**Pull request by gsk967**
## Description

closes: #1226 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -152,17 +152,40 @@ endif
...
+	@$(golangci_lint_cmd) run --fix --out-format=tab --issues-exit-code=0 --timeout=8m
+	@cd price-feeder && $(golangci_lint_cmd) run --fix --out-format=tab --issues-exit-code=0 --timeout=8m
+
+.PHONY: lint lint-fix
+
+format:
+	@go install mvdan.cc/gofumpt@latest
+	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(golangci_version)
+	find . -name '*.go' -type f -not -path "./vendor*" -not -path "*.git*" -not -path "./client/docs/statik/statik.go" -not -path "./tests/mocks/*" -not -name "*.pb.go" -not -name "*.pb.gw.go" -not -name "*.pulsar.go" -not -path "./crypto/keys/secp256k1/*" | xargs gofumpt -w -l
+	$(golangci_lint_cmd) run --fix
```
Comment (robert-zaremba) [inline Makefile:179]: This is already run in `lint-fix`. Let's don't duplicate. Instead we can add it as dependency, or just keep it only in `lint-fix`. I would stick with running `golangci-lint` only in `lint-fix`.

Comment (robert-zaremba) [reply Makefile:179]: or maybe we can move gofumpt to lint-fix?

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -7,6 +7,26 @@ on:
...
+            **/*.go
+      - uses: actions/setup-go@v3
+        if: env.GIT_DIFF
+        with:
+          go-version: 1.19
+          cache: true
+      - name: revive lint action
+        uses: morphy2k/revive-action@v2
+        with:
+          path: "./..."
```
Comment (robert-zaremba) [inline .github/workflows/lint.yml:28]: how about adding it under golangci job? This will save us some CI cycles.

File: Makefile (hunk 1 above)
Comment (gsk967) [reply Makefile]: Okay moving gofumpt and into `lint-fix`

File: .github/workflows/lint.yml (hunk 2 above)
Comment (gsk967) [reply .github/workflows/lint.yml:28]: Okay i will move revive lint into golangci job

**Pull request merged**